### DISCUSSION TITLE
Will all users always be shown the certificate window?

### DISCUSSION THREAD

**Discussion by Alik2015** [simpleidserver/SimpleIdServer#605] [General]
I am wondering when I get this working how I need it and implement in my project, will there always be the certificate selection popup even in live environment?

How can I switch off the popup or what is the alternative to it, do I need it?  I am new to this project and just trying out for a new project so just want to make sure it is the right solution for me.

Appreciate any guidance on where to learn more or what to look for.

**Comment by simpleidserver** [comment]
Hello,

By default, SimpleIdServer is configured to accept a `Client Certificate` from your browser, which is required when the client is using the authentication methods `tls_client_auth` or `self_signed_tls_client_auth`. 
Currently, it is not possible to disable this feature via the configuration file. However, you can do so by following these steps:

1. Follow the tutorial to install a local version: https://simpleidserver.com/docs/installation#create-identityserver-project.
2. Edit the `Program.cs` file.
2. Comment out the line `o.ClientCertificateMode = ClientCertificateMode.AllowCertificate` and start the application.

There is a ticket #590 for the next release 4.0.5, and in the future, it will be possible to enable or disable this feature using the configuration file.

**Comment by simpleidserver** [comment]
The ticket #590 is finished and will be published with the release `4.0.5`.
